pkg/authorization/authorizer/authorizer.go

```diff
@@ -52,7 +52,14 @@ func (a *openshiftAuthorizer) Authorize(ctx kapi.Context, passedAttributes Autho
 		return false, "", kerrors.NewAggregate(errs)
 	}
 
-	return false, "denied by default", nil
+	user, _ := kapi.UserFrom(ctx)
```

Comment:
not sure what the behavior of user.GetName() is if UserFrom says it didn't exist